testsuite/cluster-test/src/cluster_swarm/templates/vault_spec_template.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       securityContext:
         capabilities:
           add: ["IPC_LOCK"]
-      image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/vault:1.4.0
```

Comment:
May I ask the purpose we changed image pulling from ecr back to dockerhub?

Reply:
Looks like a rebase gone bad. This should have been caught during the review

Reply:
oops, sorry for that, yep this must've happened during the rebase

Reply:
no worries, that happens sometimes